src/test/java/com/fasterxml/jackson/databind/struct/UnwrapSingleArrayMiscTest.java

```diff
@@ -78,4 +81,21 @@ public void testEnumMapUnwrapping() throws Exception
             verifyException(e, "more than one value");
         }
     }
+
+    @Test
+    public void testDeserializeArrayWithNullElement() throws IOException {
+        String json = "{\"value\": [null]}";
```

Comment:
Formatting + Exception is sort of convention now

```suggestion
    public void testDeserializeArrayWithNullElement()
        throws Exception
    {
        String json = "{\"value\": [null]}";
```